test/functionalTest/harnessFunctions.sh

```diff
@@ -278,6 +278,7 @@ function brokerStartAwait
     if [ "$?" == "0" ]
     then
       vMsg The orion context broker has started, listening on port $port
+      echo "Broker started after $loopNo checks" >> /tmp/brokerStart
```

Comment:
A comment should be added above this line explaining that if this line is removed then problems may occur due to unknown causes. Otherwise, we could forgot about this fix and, passed some time, removed this appartently "harmless" echo line.